src/Drawer/Drawer.js

```diff
@@ -197,8 +199,16 @@ class Drawer extends Component {
     }
   };
 
+  getTranslatedWidth() {
+    if (this.props.width <= 1) {
+      return this.props.width * window.innerWidth;
```

Comment:
@oliviertassinari Makes sense. I just saw `window.innerWidth` being used in the `<Popover/>`. If I put a guard statement in there what should be the default value of the width?